AnnoyancesFilter/sections/mobile-app.txt

```diff
@@ -335,6 +335,8 @@ hg6384.com-www.6u7u.com##.focus-wrap
 hg6384.com-www.6u7u.com##.copyright-tips
 fdoctor.ru##section.app
 kobi8.com##div[id^="dow_APP"]
+m.tokopedia.com##div[data-testid="divOverlay"]
+m.webtoons.com##div[class$="_homeAppDownloadPopup"]
```

Comment:
It's better to use class selector here for the popup, and we need to unblock scrolling. Please add these rules:

```
webtoons.com#$#._homeAppDownloadPopup { display: none!important; }
webtoons.com#$#body { overflow: visible!important; position: relative!important; }
```

Reply:
Okay.

Reply:
8249dbc55a2da08cbefa320d5304d99e2142ea84